src/metric_config.cpp

```diff
@@ -44,6 +45,13 @@ const std::string METRIC_NAME_CURRENT_REQUESTS = "ovms_current_requests";
 const std::string METRIC_NAME_REQUEST_TIME = "ovms_request_time_us";
 const std::string METRIC_NAME_WAIT_FOR_INFER_REQ_TIME = "ovms_wait_for_infer_req_time_us";
 
+// MediaPipe
+const std::string METRIC_NAME_CURRENT_GRAPHS = "ovms_current_graphs";
```

Comment:
there are mediapipes and there are pipelines(directed acyclic graph). It may be confusing which one it refer to. To discuss

Reply:
@dtrawins do you also think this metric name could be confused with DAG Graphs? I can't come up with different name since mediapipe always refers to it as graphs.

I think we are fine, we mainly refer to DAG graphs as "pipelines" in our documentation

Reply:
I think current name should be ok but to avoid any confusion we could name is ovms_current_mp_graphs